core/src/test/java/org/apache/accumulo/core/file/rfile/bcfile/CompressionTest.java

```diff
@@ -279,41 +279,35 @@ public void testThereCanBeOnlyOne() throws InterruptedException, ExecutionExcept
         ExecutorService service = Executors.newFixedThreadPool(20);
 
         final int numTasks = 40;
-        ArrayList<Callable<Boolean>> list = new ArrayList<>(numTasks);
+        ArrayList<Callable<Integer>> list = new ArrayList<>(numTasks);
 
         CountDownLatch startSignal = new CountDownLatch(numTasks);
 
         // keep track of the system's identity hashcodes.
-        final HashSet<Integer> testSet = new HashSet<>();
+        final ArrayList<Integer> hashCodes = new ArrayList<>();
```

Comment:
It seems the ArrayList is just converted to HashSet later, why not just use HashSet from the start?

Reply:
fixed in 6c12cc0